cmd/console-server/main.go

```diff
@@ -67,7 +67,7 @@ Major items that are unfinished:
 
 
 */
-func authHandler(next http.Handler) http.Handler {
+func authHandler(next http.Handler, coreClient *enmassev1beta1.EnmasseV1beta1Client) http.Handler {
```

Comment:
Passing the existing coreClient argument to this function is wrong.  The coreClient used for the mutations needs to be associated with the bearer token of the web user in order that the actions are carried out by the user (rather than the console-service service account(.  The authHandler needs to create its own coreClient.